catalogue/webapp/pages/work.js

```diff
@@ -359,7 +267,7 @@ WorkPage.getInitialProps = async (
       workType,
       itemsLocationsLocationType,
       showNewMetaDataGrouping,
-      showWorkHeader,
+      showWorkHeader: true,
```

Comment:
Is this always `true` or meant to be dynamic?

Reply:
Indeed, it isn't and thank you for catching!